### PR TITLE
Sprint Challenge: Advanced CSS - Space Walkers Web Page

### PR DESCRIPTION
**Pull request by tmshkr**

### CODE REVIEW

**Comment by tmshkr** [comment]
[Live Site](https://tmshkr.github.io/Sprint-Challenge--Advanced-CSS/)

File: README.md [hunk 1]
```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+An adaptive website will select an appropriate HTML template based on the screen size of the device being used to access the website in order to send that HTML to the client, so that the website will be displayed correctly for a specific screen size. A fully responsive website, on the other hand, will use the same HTML template for all screen sizes, dynamically adjusting the layout of the website according to the screen size of the device using `@media` queries and responsive units.
```
Comment (jskway) [inline README.md:37]: 💯

File: README.md [hunk 2]
```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
...
 
+A mobile-first design will target mobile devices first, and then expand the layout to fill larger screen sizes. This is compared to a desktop-first approach, which targets desktop browsers first and then adjusts the layout to accommodate smaller screen sizes.
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+`font-size: 62.5%` will set the base font size of the root `html` element to 62.5% of its normal size, so that `1rem` will then be equal to 62.5% of the browser's base font size. For example, if the browser's base font size is `16px`, then `font-size: 62.5%` will make `1rem` equal to `10px`.
+
 4. How would you describe preprocessing to someone new to CSS?
 
+Preprocessors like LESS and SASS offer useful features and a more robust syntax than CSS, so that stylesheets written with these tools are more powerful, specific, and organized than regular CSS. Although their syntax is similar to regular CSS, browsers can't natively use stylesheets written in LESS or SASS, so they must be compiled to regular CSS, usually using a command-line based tool.
```
Comment (jskway) [inline README.md:49]: 💯

File: index.html [hunk 3]
```diff
@@ -1,71 +1,144 @@
...
+        </label>
+        <label>
+          Email:
+          <input type="email" name="email" />
+        </label>
+        <button>Submit</button>
+      </form>
+    </div>
+    <script>
+      var modal = document.getElementById("modal");
```
Comment (jskway) [inline index.html:123]: Impressive use of JS to make the contact form!

File: less/home-page.less [hunk 4]
```diff
@@ -57,19 +67,35 @@
         margin-bottom: 26px;
       }
 
-      .btn {
+      button {
+        .button(@ebony-clay, @sea-shell, 2rem, @border-radius: 8px);
```
Comment (jskway) [inline less/home-page.less:71]: great use of variables and parametric mixin

File: less/mixins.less [hunk 5]
```diff
@@ -1 +1,29 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.border-radius (@radius: 5px) {
+  -webkit-border-radius: @radius;
+  -moz-border-radius: @radius;
+  border-radius: @radius;
+}
+
+.transition (@transition) {
```
Comment (jskway) [inline less/mixins.less:8]: Great idea to create a mixin for these browser prefixes 👍

**Review by jskway — APPROVED** [review]

**Pull request merged**